clients/java/src/main/java/io/camunda/zeebe/client/api/access/DecisionDefinitionGetXmlRequest.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.access;
```

Comment:
@tmetzke I hesitated whether this should be included under which package:`search` or `command` (like TopologyRequestStep1). I’ve created a new package called `access`, but it can be renamed to `fetch`. WDYT?

Reply:
Thanks for flagging, @houssain-barouni 👍 

**TL;DR**: I'd vote for `fetch` instead of `access`, a new package makes sense 👍 

**Longer story:**
We have 3 categories of client interactions here now: `search`, `command`, and `fetch`:
- `command` comprises all gRPC-based interactions (that can additionally support HTTP) that create commands to be processed by the engine. The Topology request is a bit of a grey area, it doesn't strictly fit into this category.
- `search` contains the queries you can POST to the gateway via HTTP to retrieve data via search.
- `fetch` holds all HTTP-based data-fetching interactions that retrieve data. The Topology request COULD fit in here semantically as well. I don't mind leaving it where it currently is though.